Client/Assets/LoginsHelper.js

```diff
@@ -599,19 +599,28 @@ function onBlur(event) {
   LoginManagerContent.onUsernameInput(event)
 }
 
-window.addEventListener("load", function(event) {
+var documentBody = document.querySelector('body')
+var observer = new MutationObserver(function(mutations) {
+    findLogins()
+});
+
+observer.observe(documentBody, { attributes: false, childList: true, characterData: false });
+
+function findLogins(event) {
   try {
     for (var i = 0; i < document.forms.length; i++) {
       LoginManagerContent._asyncFindLogins(document.forms[i], { })
```

Comment:
Yeah, I had considered the same...my initial thought was that we should be fine since `MutationObserver` is async, but perhaps we could throw in a `setTimeout` to wait for the next tick, if needed.